src/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local package" value="Caliburn.Micro.Autofac.Packaging\" />
```

Comment:
Tried with that now, and then with `update-package -reinstall -project Sample-Win8.1` but package manager console warns about this:

> Reinstalling 'Caliburn.Micro.AutofacBootstrap 2.0.9' in project 'Sample-Win8.1'.
Skipped reinstalling package 'Caliburn.Micro.AutofacBootstrap 2.0.9' in project 'Sample-Win8.1' because the package does not exist in the package source.

What am I doing wrong?